src/components/soshistorypage/HistoyFilter.tsx

```
@@ -0,0 +1,145 @@
+  const isInvalidDate =
+    startDate.yyyy &&
+    startDate.mm &&
+    startDate.dd &&
+    endDate.yyyy &&
+    endDate.mm &&
+    endDate.dd &&
+    new Date(`${endDate.yyyy}-${endDate.mm}-${endDate.dd}`) <
+    new Date(`${startDate.yyyy}-${startDate.mm}-${startDate.dd}`);
+
+  const onSubmit = (data: FormData) => {
+    console.log('필터 값:', data);
```

ESLint

[error] Unexpected console statement.

(no-console)